db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Provisioning/Naming.class/__methods__/vmname.rb

```diff
@@ -1,58 +1,40 @@
-#
-# Description: This is the default vmnaming method
-# 1. If VM Name was not chosen during dialog processing then use vm_prefix
-#    from dialog else use model and [:environment] tag to generate name
-# 2. Else use VM name chosen in dialog
-# 3. Then add 3 digit suffix to vm_name
-# 4. Added support for dynamic service naming
-#
+=begin
+  Description: This is the default naming scheme
+  1. If VM Name was not chosen during dialog processing then use vm_prefix
+     from dialog else use model and [:environment] tag to generate name
+  2. Else use VM name chosen in dialog
+  3. Add 3 digit suffix to vmname if more than one VM is being provisioned
+=end
 
 $evm.log("info", "Detected vmdb_object_type:<#{$evm.root['vmdb_object_type']}>")
-
 prov = $evm.root['miq_provision_request'] || $evm.root['miq_provision'] || $evm.root['miq_provision_request_template']
-
 vm_name = prov.get_option(:vm_name).to_s.strip
-number_of_vms_being_provisioned = prov.get_option(:number_of_vms)
-diamethod = prov.get_option(:vm_prefix).to_s.strip
-
-# If no VM name was chosen during dialog
-if vm_name.blank? || vm_name == 'changeme'
-  vm_prefix = nil
-  vm_prefix ||= $evm.object['vm_prefix']
-  $evm.log("info", "vm_name from dialog:<#{vm_name.inspect}> vm_prefix from dialog:<#{diamethod.inspect}> vm_prefix from model:<#{vm_prefix.inspect}>")
-
-  # Get Provisioning Tags for VM Name
-  tags = prov.get_tags
-  $evm.log("info", "Provisioning Object Tags: #{tags.inspect}")
+vm_name = nil if vm_name.blank? or vm_name == 'changeme'
 
-  # Set a Prefix for VM Naming
-  if diamethod.blank?
-    vm_name = vm_prefix
-  else
-    vm_name = diamethod
-  end
-  $evm.log("info", "VM Naming Prefix: <#{vm_name}>")
+# Returns the name prefix (preferences model over dialog) or nil
+def get_prefix(prov)
+  vm_name_prefix = prov.get_option(:vm_prefix).to_s.strip
+  return $evm.object['vm_prefix'] || vm_name_prefix
+end
 
```

Comment:
@01000101 
Could you move the get_prefix, get_env_tag and get _suffix methods to the top of the file?